docs-v2/reference/api-configuration.mdx

```diff
@@ -193,7 +193,7 @@ slack:
 </ResponseField>
 
 <ResponseField name="auth_mode" type="string" >
-    The authentication mode. Must be one of: "API_KEY", "APP", "APP_STORE", "BASIC", "NONE", "OAUTH1", "OAUTH2", "OAUTH2_CC", "CUSTOM", "TBA", "TABLEAU", "JWT", "BILL", "TWO_STEP".
+    The authentication mode. Must be one of: "API_KEY", "APP", "APP_STORE", "BASIC", "NONE", "OAUTH1", "OAUTH2", "OAUTH2_CC", "CUSTOM", "TBA", "TABLEAU", "JWT", "BILL", "TWO_STEP", "SIGNATURE_BASED".
```

Comment:
what about simply `SIGNATURE`

Reply:
I have updated this with the latest [commit](https://github.com/NangoHQ/nango/pull/2942/commits/b4d3dd0ffcd24f1188b251597107f09a4e90eb13)